src/contracts/pods/EigenPodManagerStorage.sol

```diff
@@ -64,6 +64,12 @@ abstract contract EigenPodManagerStorage is IEigenPodManager {
      */
     mapping(address => int256) public podOwnerShares;
 
+    /// @notice This is the offchain proving service
+    ProofService public proofService;
+
+    /// @notice Switch to turn off partial withdrawal merkle proofs and turn on offchain proofs as a service
+    bool public partialWithdrawalProofSwitch;
```

Comment:
partialWithdrawalProofServiceEnabled

Reply:
IMO just `proofServiceEnabled` is cleaner

Reply:
Gonna bump this request again - see other comments for rationale, but I want to avoid "switch"